staging/src/k8s.io/pod-security-admission/policy/checks_test.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package policy
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestValidChecks ensures that all registered checks are valid.
+func TestValidChecks(t *testing.T) {
```

Comment:
can we check this in validateChecks to make sure whatever custom checks get registered are coherent?

Reply:
I'm assuming you mean the "Ensure that all overrides map to existing checks." block? I intentionally didn't put that in validateChecks because if you're customizing the list of checks it seems reasonable to use the overriding version without the overridden version (i.e. only the restricted version without the baseline version).

Reply:
ah, ok